parlai/core/utils.py

```diff
@@ -596,6 +598,19 @@ def map_predictions(cls, predictions, valid_inds, batch_reply,
         return
 
 
+class ProbabilityDistribution(dict):
```

Comment:
I'd prefer if this were also included inside the offensive detector as a closure.